app/Template/task/dropdown.php

```diff
@@ -3,60 +3,47 @@
+            <?= $this->url->link('<i class="fa fa-external-link fa-fw"></i>' . t('Add external link'), 'TaskExternalLinkController', 'find', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
         <li>
-            <i class="fa fa-comment-o fa-fw"></i>
-            <?= $this->url->link(t('Add a comment'), 'CommentController', 'create', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
+            <?= $this->url->link('<i class="fa fa-comment-o fa-fw"></i>' . t('Add a comment'), 'CommentController', 'create', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
         <li>
-            <i class="fa fa-camera fa-fw"></i>
-            <?= $this->url->link(t('Add a screenshot'), 'TaskPopoverController', 'screenshot', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
+            <?= $this->url->link('<i class="fa fa-camera fa-fw"></i>' . t('Add a screenshot'), 'TaskPopoverController', 'screenshot', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
         <li>
-            <i class="fa fa-files-o fa-fw"></i>
-            <?= $this->url->link(t('Duplicate'), 'TaskDuplicationController', 'duplicate', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
+            <?= $this->url->link('<i class="fa fa-files-o fa-fw"></i>' . t('Duplicate'), 'TaskDuplicationController', 'duplicate', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
         <li>
-            <i class="fa fa-clipboard fa-fw"></i>
-            <?= $this->url->link(t('Duplicate to another project'), 'TaskDuplicationController', 'copy', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
+            <?= $this->url->link('<i class="fa fa-clipboard fa-fw"></i>' . t('Duplicate to another project'), 'TaskDuplicationController', 'copy', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
         <li>
-            <i class="fa fa-clone fa-fw"></i>
-            <?= $this->url->link(t('Move to another project'), 'TaskDuplicationController', 'move', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
+            <?= $this->url->link('<i class="fa fa-clone fa-fw"></i>' . t('Move to another project'), 'TaskDuplicationController', 'move', array('task_id' => $task['id'], 'project_id' => $task['project_id']), false, 'popover') ?>
         </li>
-        <?php if ($this->projectRole->canRemoveTask($task)): ?>
+        <?php if ($this->user->canRemoveTask($task)): ?>
```

Comment:
Your PR should be in sync with the current master branch. This line should not be removed.